jerry-core/mem/mem-heap.cpp

```diff
@@ -188,7 +188,7 @@ typedef struct
 /**
  * Heap
  */
-mem_heap_t mem_heap;
+mem_heap_t mem_heap __attribute__ ((section (JERRY_HEAP_SECTION_ATTR)));
```

Comment:
I would put the ifdef here. It is true, that this variable will likely go to .bss, but I would not force this.
```
#ifndef JERRY_HEAP_SECTION_ATTR
mem_heap_t mem_heap;
#else
mem_heap_t mem_heap __attribute__ ((section (JERRY_HEAP_SECTION_ATTR)));
#endif
```